lang/src/org/partiql/lang/eval/EvaluatingCompiler.kt

```diff
@@ -429,16 +430,18 @@ internal class EvaluatingCompiler(
         metas: MetaContainer): ThunkEnv {
         return thunkFactory.thunkFold(valueFactory.nullValue, metas, argThunks) { lValue, rValue ->
             val denominator = rValue.numberValue()
-            if (denominator.isZero()) {
-                err("/ by zero", ErrorCode.EVALUATOR_DIVIDE_BY_ZERO, null, false)
-            }
-            try {
-                (lValue.numberValue() / denominator).exprValue()
-            }
-            catch (e: ArithmeticException) {
-                // Setting the internal flag as true as it is not clear what
-                // ArithmeticException may be thrown by the above
-                throw EvaluationException(cause = e, internal = true)
+
+            strictnessContext.errorIf(
+                test = denominator.isZero(),
+                detailsConstructor = { ErrorDetails(metas, "/ by zero", ErrorCode.EVALUATOR_DIVIDE_BY_ZERO) }
+            ) {
+                try {
+                    (lValue.numberValue() / denominator).exprValue()
+                } catch (e: ArithmeticException) {
+                    // Setting the internal flag as true as it is not clear what
+                    // ArithmeticException may be thrown by the above
+                    throw EvaluationException(cause = e, internal = true)
```

Comment:
Hmm, shouldn't this be captured somehow in the error handling?

Reply:
So, my intention here was just to capture the div 0 case, not to change anything else about how the error handling works, and this try/catch block preserves that.

To be clear, I am strongly leaning against catching any exception and returning `MISSING` values in the case of internal exceptions or other unhandled exceptions since that can have negative consequences to runtime performance.

Reply:
There is still a problem for dealing with this case regardless--how do you intend to handle it, as an exception handler at every call-site explicitly?  Same performance cost, more code...